saleor/account/migrations/0082_auto_20231204_1419.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.22 on 2023-12-04 14:19
+
+from django.db import migrations
+
+
+CREATE_INDEX = """
+    CREATE INDEX CONCURRENTLY IF NOT EXISTS
+        email_upper_idx ON account_user (upper(email));
+"""
+
+
+class Migration(migrations.Migration):
+    atomic = False
+    dependencies = [
+        ("account", "0081_update_user_is_confirmed"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=CREATE_INDEX,
+            reverse_sql="DROP INDEX IF EXISTS email_upper_idx;",
```

Comment:
`DROP INDEX` also supports [CONCURRENTLY](https://www.postgresql.org/docs/11/sql-dropindex.html), this way we won't lock selects, inserts, updates, and deletes on the table (the drop index will wait until conflict are finished).

Reply:
Good catch 👍